### PR TITLE
METAL-803: Update vendoring for baremetal-operator

### PR DESCRIPTION
**Pull request by derekhiggins**
Required for RAID config

Removes local_gb from terraform properties as its not required and no longer part of the BMO api.

Sets to cpu_arch to x86_64 as this is all that is supported by the installer(and its no longer part of the BMO api).

### CODE REVIEW

File: pkg/tfvars/baremetal/baremetal.go [hunk 1]
```diff
@@ -156,8 +156,7 @@ func TFVars(numControlPlaneReplicas int64, libvirtURI, apiVIP, imageCacheIP, boo
 
 		// Properties
 		propertiesMap := map[string]interface{}{
-			"local_gb":     profile.LocalGB,
-			"cpu_arch":     profile.CPUArch,
+			"cpu_arch":     "x86_64",
```
Comment (derekhiggins) [inline pkg/tfvars/baremetal/baremetal.go:159]: I this a step backwards? I couldn't find in install-config where arch is so came to the conclusion its not supported for day1 ?

Comment (dtantsur) [reply pkg/tfvars/baremetal/baremetal.go:159]: Even better, you can use `bmh.Spec.Architecture` (and omit the value if it's empty).

**Comment by dtantsur** [comment]
/lgtm

We're unable to get rid of the CPU architecture just yet, let's follow-up.

The virtualmedia job failure looks unrelated to me.

**Pull request merged**